examples/loader-census/src/index.md

```diff
@@ -0,0 +1,57 @@
+# Census boundaries
+
+Here’s how to fetch high-resolution shapefiles from the [U.S. Census Bureau](https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html) and convert them into a web-friendly format in a bash data loader.
+
+First, you’ll need to install a few packages:
+
+```sh
+npm install shapefile topojson-client topojson-server topojson-simplify
+```
+
+Next, here’s a bash script, `ca.json.sh`:
+
+```bash
+# Download the ZIP archive from the Census Bureau (if needed).
+if [ ! -f src/.observablehq/cache/cb_2023_06_cousub_500k.zip ]; then
+  curl -o src/.observablehq/cache/cb_2023_06_cousub_500k.zip 'https://www2.census.gov/geo/tiger/GENZ2023/shp/cb_2023_06_cousub_500k.zip'
+fi
+
+# Unzip the ZIP archive to extract the shapefile.
+unzip -oqd src/.observablehq/cache src/.observablehq/cache/cb_2023_06_cousub_500k.zip
+
+# Convert the shapefile to GeoJSON, then to TopoJSON, simplify, and merge counties.
+shp2json --encoding utf-8 -n src/.observablehq/cache/cb_2023_06_cousub_500k.shp > src/.observablehq/cache/cb_2023_06_cousub_500k.json
```

Comment:
This is `.ndjson` here technically, right? (Because of `-n`?)

Reply:
it is!